docker/bin/entrypoint.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
-
-source /venv/bin/activate
```

Comment:
isn't this needed any more? Is this replaced by something else?

Reply:
I don't think we were using it anywhere!

Reply:
I'm pretty sure that's coming from inside the mariadb image (especially since it's called `docker-entrypoint.sh` not just `entrypoint.sh`)